### PR TITLE
Create CREAM high transaction volume agent

### PR DESCRIPTION
**Pull request by dikel**

### CODE REVIEW

File: cream-activity/src/agent.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          findings.push(createAlert(from, eventSig, count))
+          txCounter.reset(from, eventSig)
+        }
+      })
+    })
+
+    function createAlert(from, eventSig, count) {
+      return Finding.fromObject({
+        name: "High Transaction Activity",
+        description: `${from} did ${getEventName(eventSig)} 5 times in the last minute`,
```
Comment (dikel) [inline cream-activity/src/agent.js:31]: Not sure if these are the best name and description.

File: cream-activity/src/transaction-counter.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+//    0xaddr1: [
+//      { hash1, time1 },
+//      { hash2, time2 }
+//    ],
+//    0xaddr2: [
+//      { hash1, time1 },
+//      { hash2, time2 }
+//    ]
+//  },
+// }
```
Comment (stoqnkpL) [inline cream-activity/src/transaction-counter.js:21]: nit: remove comment

File: cream-activity/src/agent.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        const count = txCounter.increment(from, eventSig, txHash, blockTimestamp)
+
+        if (count === countThreshold) {
+          findings.push(createAlert(from, eventSig, count))
+          txCounter.reset(from, eventSig)
+        }
+      })
+    })
+
+    function createAlert(from, eventSig, count) {
```
Comment (stoqnkpL) [inline cream-activity/src/agent.js:28]: isn't count always 5?

**Review by stoqnkpL — APPROVED** [review]
LGTM!

**Pull request merged**